integration-tests/push-rpms-to-pulp/resources/managed/ec-policy.yaml

```diff
@@ -0,0 +1,27 @@
+---
+apiVersion: appstudio.redhat.com/v1alpha1
+kind: EnterpriseContractPolicy
+metadata:
+  name: standard-${component_name}
+  labels:
+    originating-tool: "${originating_tool}"
+spec:
+  description: >-
+    Includes rules for levels 1, 2 & 3 of SLSA v0.1.
+  publicKey: "k8s://openshift-pipelines/public-key"
+  sources:
+    - name: Release Policies
+      data:
+        - github.com/release-engineering/rhtap-ec-policy//data
+        - oci::quay.io/konflux-ci/tekton-catalog/data-acceptable-bundles:latest
+      policy:
+        - oci::quay.io/enterprise-contract/ec-release-policy:konflux
+      volatileConfig:
+        exclude:
+          - value: cve.cve_blockers
+            effectiveUntil: "2025-02-01T00:00:00Z"
```

Comment:
this has already passed.. what is the use of it here? I am not familiar with this field